first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,603 @@
+    "\n",
+    "This notebook takes less than a minute to run, which will typically cost less than $0.01 of compute time on your cloud environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Setup\n",
+    "<a id=\"setup\"></a>\n",
+    "\n",
+    "Run the cell below to capture the ID of the current workspace. You'll use this value to return to the current workspace after you've created a new workspace as part of the process of creating a data collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import ipywidgets as widgets\n",
+    "import subprocess\n",
+    "import widget_utils as wu\n",
+    "import vwb_folder_utils as vfu\n",
+    "from datetime import date\n",
+    "\n",
+    "'''\n",
+    "Resolves ID of current workspace.\n",
+    "'''\n",
+    "def get_current_workspace_id():\n",
+    "    CURRENT_WORKSPACE_ID_CMD_OUTPUT = !terra workspace describe --format=json | jq --raw-output \".id\"\n",
+    "    CURRENT_WORKSPACE_ID = CURRENT_WORKSPACE_ID_CMD_OUTPUT[0]\n",
+    "    return CURRENT_WORKSPACE_ID\n",
+    "\n",
+    "CURRENT_WORKSPACE_ID = get_current_workspace_id()\n",
+    "print(f\"Current workspace ID is {CURRENT_WORKSPACE_ID}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Create a new data collection\n",
+    "<a id=\"create-new-dc\"></a>\n",
+    "\n",
+    "Before creating your data collection, consider the following:\n",
+    "1. What data do you want to share? What type of resources--Cloud Storage buckets or objects, BigQuery tables, GitHub repositories--will be made available via this data collection?\n",
+    "1. With whom do you wish to share this data? Will you be sharing the data collection with all members of an existing Verily Workbench group (e.g. for your organization or team), or will you need to create a new Verily Workbench group in order to restrict access to the data collection?\n",
+    "1. Will you update the data collection by releasing future versions? What versioning scheme is most appropriate?\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "<b>Note:</b> \n",
+    "    If you'd like to restrict access to your data collection to members of a specific group, contact <a href=\"mailto:workbench-support@verily.com\">Verily Workbench Support</a> <b>before</b> creating your workspace so that a <a href=\"https://terra-docs.api.verily.com/docs/https://terra-docs.api.verily.com/docs/reference/glossary/#policy\">group policy constraint</a> can be applied at the time of workspace creation. See <a href=\"../creating_a_group.ipynb\">../creating_a_group.ipynb</a> for details on how to create a Verily Workbench group that can be used for group policy constraints on workspaces and data collections. </div>"
```

Comment:
I don't believe we need to guide users to contact support anymore for this. You can now set the policies for the data collection’s backing workspace at creation time through the UI. Can we also set workspace policies through the UI? If yes, we should update the notebook to do so. If no, we should guide users to create the workspace with policy through the UI. And then file a ticket to add CLI support for workspace policy.

Reply:
I just realized that what I said above applies only for **group** policies. For **regional** policies, we'd still need to be contacted.

Reply:
That is correct. I think, therefore, we should still include the link to support here in addition to instructions for adding the group membership constraint in the UI.